contrib/sssd.spec.in

```diff
@@ -43,6 +43,12 @@
 %global build_passkey 0
 %endif
 
+%if 0%{?fedora} >= 42 || 0%{?rhel} >= 10
```

Comment:
I don't think we need a transition period in Fedora.
Let's stop shipping old helper already in current Rawhide (i.e. f-41)

Reply:
We can add a heads up in f-41 RNs.

Reply:
I think a transition, of even just one release, is a good thing. In particular because it doesn't add any complexity (all the required code is already written and needed for RHEL).

Reply:
Fair enough. I changed to F41.

Reply:
And this helped to reveal a multihost test that depends on old helper:
https://github.com/SSSD/sssd/actions/runs/8098895062/job/22135065527?pr=7223#step:8:1020

Reply:
@shridhargadekar says it's `sssd/src/tests/multihost/basic/test_basic.py`

Reply:
I updated an AD test that would have failed (although it was not being executed), and updated the containers to use the new configuration if the new tool is present. This change has a separate commit so that it can be easily reverted once the whole tool chain is updated (including `ipa-client-install`).